drivers/hmis_csv_importer/app/views/hmis_csv_importer/import_overrides/new.haml

```diff
@@ -0,0 +1,18 @@
+= content_for :crumbs do
+  = link_to hmis_csv_importer_data_source_import_overrides_path(data_source_id: @data_source) do
+    &laquo; #{@data_source.name} Overrides
+
+%h1 Add Override for #{@data_source.name}
+= simple_form_for :override, url: hmis_csv_importer_data_source_import_overrides_path(@data_source) do |f|
+  = f.error_notification
+  .row
+    .col-md-6
+      .well
+        .form-inputs
+          = f.input :file_name, collection: HmisCsvImporter::ImportOverride.available_classes, label: 'File', as: :select_two, include_blank: false
+          = f.input :matched_hud_key, label: 'Matched HMIS ID', hint: 'Given the chosen table, should we only replace data in rows with this matching ID? If blank, all rows will be replaced.  For example if you would like all enrollments to have the same EnrollmentCoC, leave this blank.  If you would like to change the type of a single project, enter the value of the ProjectID for that project here.', required: false
+          = f.input :replaces_column, label: 'Column to replace', hint: 'EnrollmentCoC or ProjectID in the above examples.'
+          = f.input :replaces_value, label: 'Value to replace', hint: 'Leave this blank to replace the column on all matching rows.  If you only want to replace one value, for example to change all 99s in ContinuumProject to 0, since 99 is not allowed, enter 99 here.', required: false
+          = f.input :replacement_value, label: 'Replacement value', hint: 'Enter the value you would like instead of the exsiting value here.  In the above example for ContinuumProject, enter 0.'
+        .form-actions
+          = f.button :submit, class: 'btn btn-primary', value: 'Add Override'
```

Comment:
In other projects, I try and add `data-disable-with="Loading…"` from UJS on form buttons to avoid duplicate submissions from double clicks. Looks like we don't use that pattern in the warehouse though?